spikeextractors/recordingextractor.py

```diff
@@ -539,25 +539,23 @@ def copy_channel_properties(self, recording, channel_ids=None):
             The list (or single value) of channel_ids for which the properties will be copied
         '''
         if channel_ids is None:
-            self._key_properties = deepcopy(recording._key_properties)
```

Comment:
@colehurwitz this is much faster and I don't think it's harmful. In order to fix the issue, I think we can just avoid calling `copy_channel_property` from the `MultiRecordingChannelExtractor` or override the function.

Reply:
It actually works with the `MultiRecordingChannelExtractor` now since all properties are stored in their respective dictionaries. 

I was more concerned about skipping the `set_channel_property` function since it is important for key properties like location and groups. Is the speed upgrade when doing deepcopys very noticeable? I thought the speed increase would be basically negligible since channel properties are always pretty small.

Reply:
@colehurwitz  calling the `set_channel_property` for all `key_properties` and `properties` is the same as deepcopying the dictionaries. When you have several properties of >1000 channels the deepcopy is way faster. Can you add this back?

Reply:
I am fine with having deepcopy back in the code.